### PR TITLE
feat(storage): add PublicAccessPrevention samples

### PR DESCRIPTION
**Pull request by tritone**
Adds samples for 3 new region tags for PublicAccessPrevention
feature. See library PR at googleapis/google-cloud-go#3608

### CODE REVIEW

File: storage/buckets/get_public_access_prevention.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"cloud.google.com/go/storage"
+)
+
+// getPublicAccessPrevention gets the current public access prevention setting
+// for the bucket.
```
Comment (shaffeeullah) [inline storage/buckets/get_public_access_prevention.go:28]: Maybe also note the setting possibilities?
```suggestion
// for the bucket, either 'enforced' or 'unspecified'.
```

**Review by shaffeeullah — APPROVED** [review]
Looks good!! One minor suggestion

File: storage/buckets/get_public_access_prevention.go (hunk 1 above)
Comment (tritone) [reply storage/buckets/get_public_access_prevention.go:28]: Done.

File: storage/buckets/buckets_test.go [hunk 2]
```diff
@@ -353,6 +353,56 @@ func TestUniformBucketLevelAccess(t *testing.T) {
...
+	if err := setPublicAccessPreventionUnspecified(ioutil.Discard, bucketName); err != nil {
+		t.Errorf("setPublicAccessPreventionUnspecified: %v", err)
+	}
+	// Verify that PublicAccessPrevention was set correctly.
+	attrs, err = client.Bucket(bucketName).Attrs(ctx)
+	if err != nil {
+		t.Fatalf("Bucket(%q).Attrs: %v", bucketName, err)
+	}
+	if attrs.PublicAccessPrevention != storage.PublicAccessPreventionUnspecified {
+		t.Errorf("PublicAccessPrevention: got %v, want %v", attrs.PublicAccessPrevention.String(), storage.PublicAccessPreventionUnspecified.String())
```
Comment (codyoss) [inline storage/buckets/buckets_test.go:401]: nit: Looks like these will have a string method so I don't think you need to call it explicitly. At least if you use %s.

File: storage/buckets/get_public_access_prevention.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	defer client.Close()
+
+	ctx, cancel := context.WithTimeout(ctx, time.Second*10)
+	defer cancel()
+
+	attrs, err := client.Bucket(bucketName).Attrs(ctx)
+	if err != nil {
+		return fmt.Errorf("Bucket(%q).Attrs: %v", bucketName, err)
+	}
+	fmt.Fprintf(w, "Public access prevention is %v for %v", attrs.PublicAccessPrevention.String(), bucketName)
```
Comment (codyoss) [inline storage/buckets/get_public_access_prevention.go:45]: nit: use %s

File: storage/buckets/buckets_test.go (hunk 2 above)
Comment (tbpg) [reply storage/buckets/buckets_test.go:401]: No need to use `%s` if there is a `String` method. See https://play.golang.org/p/yuXkuHhSQsR.

**Review by tbpg — COMMENTED** [review]
Looks good. Will wait to approve until ready to submit.

**Review by codyoss — APPROVED** [review]
LGTM

**Review by frankyn — APPROVED** [review]

**Comment by shaffeeullah** [comment]
Public access prevention rollout has been delayed due to a bug surfaced during Googler preview. I will keep this PR updated as I learn new release timeline details.

**Pull request merged**